crates/wysiwyg/src/tests/test_paragraphs.rs

```diff
@@ -429,3 +429,18 @@ fn pressing_enter_at_the_start_of_a_multiline_block_quote() {
         "<blockquote><p>&nbsp;</p><p>|line_1</p><p>line_2</p></blockquote>"
     )
 }
+
+#[test]
+fn pressing_enter_in_the_middle_of_a_multiline_code_block() {
+    let mut model = cm("<pre>\n\n|line_1\nline_2</pre>");
+    model.enter();
+    assert_eq!(tx(&model), "<pre>&nbsp;\n\n|line_1\nline_2</pre>")
+}
+
+#[test]
+fn pressing_enter_in_the_middle_of_a_multiline_block_quote() {
+    let mut model =
+        cm("<blockquote><p></p><p>|line_1</p><p>line_2</p></blockquote>");
+    model.enter();
+    assert_eq!(tx(&model), "<blockquote><p>&nbsp;</p><p>&nbsp;</p><p>|line_1</p><p>line_2</p></blockquote>")
+}
```

Comment:
I would actually also write the test case for `<blockquote><p>line_0</p><p>|line_1</p><p>line_2</p></blockquote>` and `<pre>line_0\n|line_1\nline_2</pre>`

Reply:
I added some tests in [`f0bc508`](https://github.com/matrix-org/matrix-rich-text-editor/pull/531/commits/f0bc50822e09e4d02f7140d7c90c181f61623eb0). Let me know if those are what you meant.